modules/gitops-additional-permissions-for-cluster-config.adoc

```diff
@@ -7,11 +7,6 @@
 = Adding permissions for cluster configuration
 
 You can grant permissions for an Argo CD instance to manage cluster configuration. Create a cluster role with additional permissions and then create a new cluster role binding to associate the cluster role with a service account. 
-
-.Procedure
-
-. Log in to the {product-title} web console as an admin.
-. In the web console, select **User Management** -> **Roles** -> **Create Role**. Use the following `ClusterRole` YAML template to add rules to specify the additional permissions.
```

Comment:
Just curious about why this was removed as part of this PR; I didn't think it's mentioned in https://issues.redhat.com/browse/OSDOCS-4920 ? I could be missing something though

Reply:
It's a commit error. Needs to be separated out.

Reply:
removed.